llmfoundry/command_utils/__init__.py

```diff
@@ -26,4 +37,13 @@
     'eval_from_yaml',
     'convert_dataset_hf',
     'convert_dataset_hf_from_args',
+    'convert_text_to_mds',
+    'convert_text_to_mds_from_args',
+    'maybe_create_object_store_from_uri',
+    'parse_uri',
+    'download_and_convert',
```

Comment:
I don't think all these helpers should be exported as part of the command_utils module